Results: figure 4 (holdout cancer types analysis) (#6)
greenelab/generalization-manuscript: content/02.main-text.md

**E.** _PIK3CA_ mutation status prediction performance generalizing from TCGA (holdout, orange), to CCLE (green), with "best" and "smallest good" models labeled.
](images/figure_3.png){#fig:tcga_ccle_smallest_best width="90%"}

### Generalization across cancer types yields similar results to generalization across datasets

To evaluate generalization across biological contexts within a dataset, we trained mutation prediction classifiers on all but one cancer type in TCGA, performed model selection on a holdout set stratified by cancer type, and held out the remaining cancer type as a test set.
We performed the same "best" vs. "smallest good" analysis that was previously described, across 294 gene/holdout cancer type combinations (Figure {@fig:cancer_type_holdout}A).

> We observed 127/294 gene/cancer type combinations (43.1%) that had better generalization performance with the "best" model, compared to 78/294 (26.5%) for the "smallest good" model, and the other 89 gene/cancer type combinations having the same "best" and "smallest good" model and thus no difference in performance.

Reviewer: ```suggestion
We observed 127/294 gene/cancer type combinations (43.1%) that had better generalization performance with the "best" model, compared to 78/294 (26.5%) for the "smallest good" model.
The other 89 gene/cancer type combinations had the same "best" and "smallest good" model and thus no difference in performance.
```

> This is similar to what we observed for the cross-dataset experiments, with slightly more instances where the "best" model on the stratified holdout data also generalizes the best, but no pronounced distributional shift in either direction.

Reviewer: ```suggestion
This is consistent with our cross-dataset experiments, with slightly more instances where the "best" model on the stratified holdout data also generalizes the best, but no pronounced distributional shift in either direction.
```
